e2e/pages/constants.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+# in ms, here 30 seconds
```

Comment:
should be in documentation

Reply:
Ups, my bad, somehow lost it between rebasing my commits.

Reply:
Ah no, I just thought hinting like this `For type details and defaults see [conftest.py](conftest.py)` Is better than duplicating the values.